packages/@webex/plugin-meetings/src/member/index.ts

```diff
@@ -10,7 +10,7 @@ import MemberUtil from './util';
  * @class Member
  */
 export default class Member {
-  associatedUser: any;
+  associatedUsers: Set<string>; // member ids of users associated with this device, empty if this member is not a device
```

Comment:
Should this be participant ids or member ids?

this makes sense to me otherwise

Reply:
Should we change the type of string to MemberId or ParticipantId and have that be a representation of a string?

Reply:
MemberId is always set to ParticipantId, so they're the same. Overall I think the idea is that SDK abstracts away Locus participants and provides SDK Members instead, right? so I think fields in Member class should use MemberIds.

Sure, I can add type for MemberId, just be aware it kind of gives you a false sense of type safety, because typescript will still handle this as a string, so if you call a function that takes MemberId argument with some other string instead of MemberId, you won't get any error or warning from typescript. But having MemberId will still be useful for a developer reading the function declaration, so I'll add it.